seatunnel-engine/seatunnel-engine-common/src/main/resources/hazelcast.yaml

```
@@ -32,7 +27,11 @@ hazelcast:
       auto-increment: true
       port-count: 100
       port: 5801
-  properties:
-    hazelcast.invocation.max.retry.count: 20
-    hazelcast.tcp.join.port.try.count: 30
-    hazelcast.logging.type: log4j2
+  map:
+    map-name-template:
+      map-store:
+        enabled: true
+        initial-mode: EAGER
+        class-name: org.apache.seatunnel.engine.server.persistence.FileMapStore
+        properties:
+          path: /tmp/file-store-map
```

We only need to persist part of the IMap.
It should be like this.
```suggestion
  map:
    engine*:
      map-store:
        enabled: true
        initial-mode: EAGER
        factory-class-name: org.apache.seatunnel.engine.server.persistence.FileMapStoreFactory
        properties:
          namespace: /tmp/seatunnel/imap
          clusterName: seatunnel-clsuter
          fs.defaultFS: file:///
```

---

The IMap we need to persist should have a uniform prefix, like `engine_runningJobInfo`

---

> The IMap we need to persist should have a uniform prefix, like `engine_runningJobInfo`

Thanks, I will fix it.

---

done

---

> The IMap we need to persist should have a uniform prefix, like `engine_runningJobInfo`

@ic4y  PTAL

---

The FileMapStoreFactory has not been merged into #3637, so there is no need to add map-store configuration at this pr. @liugddx